packages/core/src/Chip/ChipContent/ChipContent.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space, fontSize } from 'styled-system'
+import { ArrowRight } from 'pcln-icons'
+import { ChipContentWrapper } from '../ChipContentWrapper'
+import { Flex } from '../../Flex'
+import { Text } from '../../Text'
+
+const ImageWrapper = styled(Flex)`
+  background-color: white;
+  border-radius: 6px;
+  ${({ disabled }) => disabled && 'opacity: .33;'}
+`
+
+const evaluateFontSize = (size) => (size === 'md' ? 1 : 0)
+
+const evaluateChipHeight = (size) => (size === 'md' ? '40px' : '32px')
+
+const evaluatePadding = (size) => (size == 'md' ? '16px' : '8px')
+
+const getSize = (size, evaluate) =>
+  typeof size === 'string' ? evaluate(size) : size.map((s) => evaluate(s))
+
+const ChipContent = ({
+  disabled,
+  selected,
+  children,
+  label,
+  Icon,
+  facet,
+  action,
+  image,
+  size,
+  bridgeLabel,
+  ...props
+}) => {
+  const fontSize = getSize(size, evaluateFontSize)
+
+  return (
+    <ChipContentWrapper
+      height={!children && getSize(size, evaluateChipHeight)}
+      px={getSize(size, evaluatePadding)}
+      disabled={disabled}
+      selected={selected}
+      size={size}
+      {...props}
+    >
+      {children}
+      {!!image && <ImageWrapper disabled={disabled}>{image}</ImageWrapper>}
```

Comment:
I think it would be better to replace all instances of `!!` with `Boolean(` so that you only have to perform one operation instead of 2.

Reply:
Updated